### PR TITLE
Sec officers always like donuts

### PR DESCRIPTION
**Pull request by Shadowlight213**
Makes security officers always like donuts

Idea from https://tgstation13.org/phpBB/viewtopic.php?f=9&t=30&start=3150#p378475

:cl:
tweak: Roundstart or latejoin sec officers will like donuts regardless of race.
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
Could be argued this is taking away creativity from players and trying to railroad personality in roles even more. It's probably fine though, considering our tastes are all unrealistically identical ingame anyways.

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 1]
```diff
@@ -23,6 +23,22 @@
 		bonus_reagents = list("sprinkles" = 2, "sugar" = 1)
 		filling_color = "#FF69B4"
 
+
+/obj/item/reagent_containers/food/snacks/donut/checkLiked(fraction, mob/M)	//Sec officers always love donuts
+	if(last_check_time + 50 < world.time)
+		if(ishuman(M))
+			var/mob/living/carbon/human/H = M
+			if(!H.has_trait(TRAIT_AGEUSIA))
+				if(H.mind && H.mind.assigned_role == "Security Officer")
```
Comment (lordpidey) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]: Can you merge these two if statements?

**Comment by Iamgoofball** [comment]
@Bawhoppen Security Officers getting bonuses from donuts has always been a thing in the codebase, this just extends it.

**Comment by Bawhoppen** [comment]
I am aware of that, but that seems kind of different than this. Anyways it really doesn't matter this time though.

**Pull request merged**